### PR TITLE
Change copybutton

### PR DESCRIPTION
**Pull request by leminchan**
Added clipboard.js (MIT Licensing) files;
Tested mobile and Chrome responsiveness

Signed-off-by: Mit Chan <leminc@uci.edu>

### CODE REVIEW

File: _includes/code_snippet.md [hunk 1]
```diff
@@ -5,19 +5,39 @@
...
+    // Set Button HTML to 'Copied!'
+    document.querySelector('#copybutton{{ nanosecond }}').innerHTML = 'Copied!';
+
+    // After 2 seconds, return back to original text.
+    setTimeout(() => {
+     document.querySelector('#copybutton{{ nanosecond }}').innerHTML = 'Copy to clipboard';
+    }, 2000);
+
+    /* Testing custom logic through console
+    console.info('Action:', e.action);
```
Comment (joewxboy) [inline _includes/code_snippet.md:26]: I understand needing the console.info lines for testing, but are they needed in production?

**Comment by joewxboy** [comment]
@leminchan Were there any other pages that also used the clipboard copying functionality?  I have a vague idea that we used that on one other page.

**Comment by joewxboy** [comment]
@leminchan One other question ... do we have a corresponding issue to tie to the PR?

**Comment by leminchan** [comment]
I think there are three pages using clipboard copying functionality: quick-start.md, use.md, and contribute.md.

I just checked existing issues, but I don't believe there is a corresponding issue to tie to this PR.

**Review by joewxboy — APPROVED** [review]
Approved.

**Pull request merged**